ovh/provider.go

```diff
@@ -91,6 +91,7 @@ func Provider() *schema.Provider {
 			"ovh_cloud_project_user_s3_credentials":                   dataCloudProjectUserS3Credentials(),
 			"ovh_cloud_project_user_s3_policy":                        dataCloudProjectUserS3Policy(),
 			"ovh_cloud_project_users":                                 datasourceCloudProjectUsers(),
+			"ovh_dbaas_logs":                                          dataSourceDbaasLogs(),
```

Comment:
The datasource should be renamed to `ovh_dbaas_logs_cluster`:)

Reply:
done